open-metadata-implementation/repository-services/repository-services-apis/src/main/java/org/odpi/openmetadata/repositoryservices/connectors/stores/metadatacollectionstore/OMRSMetadataCollection.java

```diff
@@ -1903,6 +2007,31 @@ public abstract EntityDetail declassifyEntity(String  userId,
                                                                                      UserNotAuthorizedException,
                                                                                      FunctionNotSupportedException;
 
+    /**
+     * Remove a specific classification from an entity proxy
+     *
+     * @param userId unique identifier for requesting user.
+     * @param entityGUID String unique identifier (guid) for the entity.
+     * @param classificationName String name for the classification.
+     *
+     * @return EntityProxy showing the resulting entity header, properties and classifications.
+     *
+     * @throws InvalidParameterException one of the parameters is invalid or null.
+     * @throws RepositoryErrorException there is a problem communicating with the metadata repository where
+     *                                  the metadata collection is stored.
+     * @throws EntityNotKnownException the entity identified by the guid is not found in the metadata collection
+     * @throws ClassificationErrorException the requested classification is not set on the entity.
+     * @throws UserNotAuthorizedException the userId is not permitted to perform this operation.
+     * @throws FunctionNotSupportedException the repository does not support maintenance of metadata.
+     */
+    public abstract EntityProxy declassifyEntityProxy(String  userId,
```

Comment:
similar comment - should be declassifyEntity with entity proxy replacing entityGUID